index.html

```diff
@@ -899,57 +574,512 @@ <h3><dfn>RTCQuicTransportState</dfn> Enum</h3>
       </div>
     </section>
   </section>
+  <section id="quic-transport*">
+    <h2><dfn>QuicTransport</dfn> Interface</h2>
+    <p>The <code>QuicTransport</code> is a subclass
+    of <code>QuicTransportBase</code> focused on client/server use
+    cases.</p>
+    <section id="quictransport-operation*">
+      <h3>Operation</h3>
+      <p>A <code><a>QuicTransport</a></code> instance is constructed
+      using an URL which is constrained by CORS, just like a WebSocket.</p>
+    </section>
+    <section id="quictransport-interface-definition*">
+      <h3>Interface Definition</h3>
+      <div>
+        <pre class="idl">
+        [ Constructor (DOMString url), Exposed=Window]
+interface QuicTransport : QuicTransportBase {
+};</pre>
+        <section>
+          <h2>Constructors</h2>
+          When the <code><a>QuicTransport</a></code> constructor is invoked,
+          the user agent <em class="rfc2119" title="MUST">MUST</em> run the
+          following steps:
+          <ol>
+            <li>Let <var>url</var> be the first argument.</li>
+            <li>Let <var>parsedUrl</var> be the result of appyling the 
+              <a href="https://url.spec.whatwg.org/#concept-url-parser">URL parser</a> to <var>url<var>.
+            </li>
+            <li>If <var>parsedUrl</var> is a failure, then throw a "SyntaxError" DOMException.</li>
+            <li>If <var>parsedUrl</var>'s scheme is not "quic", then throw a "SyntaxError" DOMException.</li>
+            <li>If <var>parsedUrl</var>'s username, password, path, query, or fragment is non-null,
+              then throw a "SyntaxError" DOMException.</li>
+            <li>
+              Let <var>quictransport</var> be a newly constructed
+              <code><a>QuicTransport</a></code> object.
+            </li>
+            <li>Let <var>quictransport</var> have a <dfn>[[\QuicTransportWritableStreams]]</dfn>
+            internal slot representing a sequence of <code><a>QuicWritableStream</a></code>
+            objects, initialized to empty.
+            </li>
+            <li>Let <var>quictransport</var> have a <dfn>[[\QuicTransportReadableStreams]]</dfn>

```

Comment:
The page complains with duplicate definitions of QuicTransportReadableStreams:
_"Duplicate definitions of '[[quictransportreadablestreams]]' at: 1."_